api/src/db/model/profile.ts

```diff
@@ -249,6 +249,29 @@ export default class ProfileModel extends Model {
       throw err;
     }
   }
+  
+  async removeContactFromProfile(profileId: number, contactId: number): Promise<void> {
+    const values: any[] = [];
+    const table = 'profile_contact';
+    const query = {
+      text: `DELETE FROM ${table} WHERE contact_id = $1 AND profile_id = $2 RETURNING *;`,
```

Comment:
This is something we need to clean up in the API at somepoint. It would be advantageous to pick a formatting style and stick to it. ${table} uses the variable, where $1 uses the values syntax. I recall @jleach had looked into a best practice here previously.

Reply:
I will create a separate ticket for those.